docs/glossary.md

```diff
@@ -11,3 +11,19 @@
 - **Game**: (AKA Game XML) refers to a specific XML within a map
 - **Map**: collection of files, eg: images, polygons text, properties, and at least one game XML
 
+## Battle Terms:
+- **Roll**: How many dice can the unit roll.
+- **Strength**: The number on the dice that determines whether a hit was made. Any number less than or equal to the number is a hit.
+- **Round**: A battle round includes both sides firing at each other. A battle can consist of multiple rounds. During and at the end of each round, casualties are removed from the game and can not participate anymore.
+- **Phase**: A battle phase is subpart of a battle round where specific rules are in play that determine what the units can do. Some phases allow the units to retreat. Other phases allow the units to fire at each other.
+- **General Phase**: This phase is the default phase for units to fire. The attacker fires first and then the defender retaliates. Any casualties that the attacker hits can retaliate. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle.
+- **First Strike Phase**: This phase is for units to fire and remove casualties before the General Phase. Historically, this was used for submarines to sink sea based units as a "sneak attack". Now, any unit can participate during this phase if they have the `isFirstStrike` attribute. Units that fire during this phase will not participate during the General Phase. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle.
+  - If an opposing side has an `isDestroyer` unit, this phase is skipped.
+  - WW2V2 always has this phase. The casualties will not be removed at the end of the phase if an `isDestroyer` unit is present on the opposing side.
+  - `Defending Subs Sneak Attack` game property controls whether the defender gets to fire in the phase.
+- **Targeted Attack Phase**: This phase is one of the first battle phases. Units determine their strength and roll based on their AA attributes (such as `attackAA` and `offensiveAttackAA`). If a unit has AA attributes, they can fire at this time. The `typeAA` and `targetsAA` attributes determine what units are targeted during this attack. At the end of the phase, all casualties are removed from the game or transformed into other units which do not participate in the battle. Units that fire during this phase can fire during later phases if they have normal strength and roll attributes.
```

Comment:
I would have thought this was the 'anti-aircraft fire' phase. I suspect most players would not know what the 'targeted attack phase' is. Checking if players use this language naturally is a test if the domain terminology fits. In this case I don't think it necessarily does.

Reply:
I have seen people refer to it as "targeted attacks" and as "aa attacks".  The most recent case was in the warcraft war heroes thread.  I assumed that the new terminology has been "targeted" since that includes more units than "aa".  But I really don't know.

Reply:
I've renamed it to "AA Combat Phase" and added a new line of "Targeted Attack Combat Phase" that states it is a another name for "AA Combat Phase" and when it is used instead of "AA".